awx/api/serializers.py

```diff
@@ -2981,104 +3018,160 @@ class Meta:
         fields = ('can_cancel',)
 
 
-class WorkflowNodeBaseSerializer(BaseSerializer):
-    job_type = serializers.CharField(allow_blank=True, allow_null=True, required=False, default=None)
```

Comment:
Is it possible to have an old WorkflowNode w/ a `job_type` _not in_ `NEW_JOB_TYPE_CHOICES`?

Reply:
Yep, nodes that exist prior to 3.2.